docs/docs/dev-manuals/services/discount.mdx

```diff
@@ -67,31 +67,28 @@ interface Order {
   userId: string,
   createdAt: Date,
   orderStatus: OrderStatus,
-  placedAt?: Date,
```

Comment:
What's the difference between `createdAt` and `placedAt`?
Can an order be created without being placed?

Reply:
createdAt = when the user starts the order process (the point in time where you are precented with the overview (immediately before you press the "order now" button)
placedAt = after you pressed the "order now" button